### ISSUE TITLE
Old Documentation on the web are way too old

### ISSUE THREAD

**Issue by GillesDuvert** [gnudatalanguage/gdl#778]
As seen from a comment in #773  people in need of documentation about GDL will try to find it by googling. Most of the time they fall on old documentation that does not give credits to the numerous improvements done since gdl 0.8.
As we know writing a comprehensive up-to-date documentation is bound to fail for 2 reasons:
- The IDL one is perfect and availble
- lack of manpower

It would be advisable to remove as far as possible any specifics about any GDL evolutionary status from the small number of sites returned by google.
Wikipedia is one of these sources and its external links should be removed (or updated) (2010!).

Help welcome!

**Comment by slayoo** [comment]
Wikipedia entry updated: https://en.wikipedia.org/w/index.php?title=GNU_Data_Language&diff=963000388&oldid=962507673

@acoulais How we can help with updating "the matrices"?
http://aramis.obspm.fr/~coulais/IDL_et_GDL/Matrice_IDLvsGDL_missing.html

**Comment by alaingdl** [comment]
Yes, I have to do that again, and maybe to distribute the code. But no time in next days.

How to tag it "to be done for 1.0.0" ?!

**Comment by slayoo** [comment]
![image](https://user-images.githubusercontent.com/89685/84919979-e330ed80-b0c2-11ea-8a2b-e9b43e388d55.png)
@acoulais I've just tagged it for the 1.0.0 release (see screenshot above)
